docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -300,8 +300,7 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 - `ENABLE_GZIP`: **false**: Enable gzip compression for runtime-generated content, static resources excluded.
 - `ENABLE_PPROF`: **false**: Application profiling (memory and cpu). For "web" command it listens on localhost:6060. For "serv" command it dumps to disk at `PPROF_DATA_PATH` as `(cpuprofile|memprofile)_<username>_<temporary id>`
 - `PPROF_DATA_PATH`: **data/tmp/pprof**: `PPROF_DATA_PATH`, use an absolute path when you start Gitea as service
-- `LANDING_PAGE`: **home**: Landing page for unauthenticated users \[home, explore, organizations, login\].
-
+- `LANDING_PAGE`: **home**: Landing page for unauthenticated users \[home, explore, organizations, login, **custom**\]. Where custom would instead be any URL such as "/org/repo" or even "https://anotherwebsite.com"
```

Comment:
![image](https://user-images.githubusercontent.com/2114189/161813824-11b5ebe1-b36f-4bae-98f8-7e7e367ace49.png)

The URL in document becomes a link on the document page, it seems not good.

Reply:
Hmm.... perhaps if I wrap it in ` that'll escape the link

Reply:
I vaguely recall a GH action for netlify previews, maybe that would be nice for PRs touching docs.
(Until we move to gitea.com, but we could find an alternative then)